tests/lvmo/test_lvm_snapshot_node_reboot.py

```diff
@@ -0,0 +1,301 @@
+import logging
+import time
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier2, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM
+from ocs_ci.ocs.exceptions import (
+    LvSizeWrong,
+    CommandFailed,
+    PodDidNotReachRunningState,
+)
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.ocp import switch_to_project
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.node import get_node_objs
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@skipif_lvm_not_installed
+@skipif_ocs_version("<4.11")
+class TestLvmSnapshotNodeReboot(ManageTest):
+    """
+    Test lvm snapshot with node reboot
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["volume_mode", "volume_binding_mode"],
+        argvalues=[
+            pytest.param(
+                *[constants.VOLUME_MODE_FILESYSTEM, constants.WFFC_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-3991"),
+            ),
+            pytest.param(
+                *[constants.VOLUME_MODE_BLOCK, constants.WFFC_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-3991"),
+            ),
+            pytest.param(
+                *[
+                    constants.VOLUME_MODE_FILESYSTEM,
+                    constants.IMMEDIATE_VOLUMEBINDINGMODE,
+                ],
+                marks=pytest.mark.polarion_id("OCS-3991"),
+            ),
+            pytest.param(
+                *[constants.VOLUME_MODE_BLOCK, constants.IMMEDIATE_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-3991"),
```

Comment:
all polarion IDs are the same

Reply:
The tests that where suppose to check this parameters have multiple polarion ID. All my other tests are parameterize this way and I'm not using different polarion ID for each parameter as it is not stated in the test plan.